src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Shapes/Shape_StrokeTest.xaml.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using Uno.UI.Samples.Controls;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.UI;
+using Windows.UI.ViewManagement;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Navigation;
+using XamlWindow = Windows.UI.Xaml.Window;
+
+namespace UITests.Windows_UI_Xaml_Shapes
+{
+	[Sample("Shapes")]
+	public sealed partial class Shape_StrokeTest : Page
+	{
+		private readonly List<SolidColorBrush> _brushes =
+			new[] { Colors.Red, Colors.Green, Colors.Blue }
+				.Select(x => new SolidColorBrush(x))
+				.ToList();
+
+		public Shape_StrokeTest()
+		{
+			this.InitializeComponent();
+			TestTarget.Stroke = _brushes[0];
+
+			// Apperently, these scenarios work without the fix..?:
+			// - OK: assign Shape.Stroke directly
+			// - OK: assign Shape.Stroke.Color directly
+			// - OK: updating Shape.Stroke.Color(ThemeResource) with (dark/light) theme change
+			// - FAIL: assign Shape.Stroke.Color directly within a Grid?
```

Comment:
The fact that the Ellipse is within a Grid or not can break this, suggests there is more to it...
Are we okay with just this fix for now?

Reply:
Let's fix this one for now.

Reply:
I should maybe clarify this... So the "OK" scenarios works, and this PR addresses the "FAIL" one